pkgs/development/libraries/libcef/default.nix

```diff
@@ -0,0 +1,84 @@
+{ stdenv, mkChromiumDerivation, fetchgit, gtkglext
+, fontconfig, freetype, cairo, pango, libX11, libXi, libXext
+, libXfixes, libXrender, libXcomposite, dbus, atk, libXrandr
+, libXmu, libXt, libSM, libICE }:
+
+with stdenv.lib;
+
+mkChromiumDerivation (base: rec {
+  name = "libcef";
+  packageName = "cef";
+  # Run "git rev-list --count HEAD" to get this number
+  cefVersion = "1414";
+  # from CHROMIUM_BUILD_COMPATIBILITY.txt
+  upstreamVersion = "51.0.2704.47";
+  version = assert base.version == upstreamVersion;
+            "${upstreamVersion}-${cefVersion}";
+
+  buildTargets = [ "mksnapshot" "chrome_sandbox" "libcef" ];
```

Comment:
It should (though looks like it still checks if `chrome-sandbox` is there, see https://bugs.chromium.org/p/chromium/issues/detail?id=598454).